pyuvm/s19_pyvum_reg_field.py

```diff
@@ -219,237 +215,238 @@ def get_response(self):
+        if self.get_access() == "WSRC":
             self._field_mirrored = _mask
-        if self.get_access() == "WCRS":  
+        if self.get_access() == "WCRS":
             self._field_mirrored = 0
-        if self.get_access() == "W1C":   
+        if self.get_access() == "W1C":
             self._field_mirrored = self._field_mirrored & (~wr_val)
-        if self.get_access() == "W1S":   
+        if self.get_access() == "W1S":
             self._field_mirrored = self._field_mirrored | wr_val
-        if self.get_access() == "W1T":   
+        if self.get_access() == "W1T":
             self._field_mirrored = self._field_mirrored ^ wr_val
-        if self.get_access() == "W0C":   
+        if self.get_access() == "W0C":
             self._field_mirrored = self._field_mirrored & wr_val
-        if self.get_access() == "W0S":   
+        if self.get_access() == "W0S":
             self._field_mirrored = self._field_mirrored | (~wr_val & _mask)
-        if self.get_access() == "W0T":   
+        if self.get_access() == "W0T":
             self._field_mirrored = self._field_mirrored ^ (~wr_val & _mask)
-        if self.get_access() == "W1SRC": 
+        if self.get_access() == "W1SRC":
             self._field_mirrored = self._field_mirrored | wr_val
-        if self.get_access() == "W1CRS": 
+        if self.get_access() == "W1CRS":
             self._field_mirrored = self._field_mirrored & (~wr_val)
-        if self.get_access() == "W0SRC": 
+        if self.get_access() == "W0SRC":
             self._field_mirrored = self._field_mirrored | (~wr_val & _mask)
-        if self.get_access() == "W0CRS": 
+        if self.get_access() == "W0CRS":
             self._field_mirrored = self._field_mirrored & wr_val
-        if self.get_access() == "WO":    
+        if self.get_access() == "WO":
             self._field_mirrored = wr_val
-        if self.get_access() == "WOC":   
+        if self.get_access() == "WOC":
             self._field_mirrored = 0
-        if self.get_access() == "WOS":  
+        if self.get_access() == "WOS":
             self._field_mirrored = _mask
-        if self.get_access() == "W1":   
-            if self._has_been_writ == True:
-                self._field_mirrored = self._field_mirrored 
+        if self.get_access() == "W1":
+            if self._has_been_writ is True:
+                self._field_mirrored = self._field_mirrored
             else:
                 self._field_mirrored = wr_val
-        if self.get_access() == "WO1":   
-            if self._has_been_writ == True:
-                self._field_mirrored = self._field_mirrored 
+        if self.get_access() == "WO1":
+            if self._has_been_writ is True:
+                self._field_mirrored = self._field_mirrored
             else:
                 self._field_mirrored = wr_val
-        if self.get_access() == "NOACCESS": 
+        if self.get_access() == "NOACCESS":
             self._field_mirrored = self._reset
 
```

Comment:
all flake8 fixes. Additionally `self._field_mirrored = self._field_mirrored` this is used to replace PASS so the mirrored value will be stabled once not needed to be changed